recipe/meta.yaml

```diff
@@ -13,7 +13,14 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install --ignore-installed --no-deps .
+  script:
+    # macOS renamed this variable, but it is `NULL` in either case.
+    # Workaround the missing definition by setting it to `0`.
+    #
+    # xref: https://developer.apple.com/documentation/iokit/kiomasterportdefault
+    # xref: https://developer.apple.com/documentation/iokit/kiomainportdefault
+    - export CLAGS="${CFLAGS} -DkIOMainPortDefault=0"  # [osx]
+    - {{ PYTHON }} -m pip install --ignore-installed --no-deps .
```

Comment:
@beckermr could you please look over this and make sure we are setting it correctly? It seems CI is still not picking up the value we set

Reply:
I have no idea.

Reply:
Ok then we might be better off bumping the minimum macOS version here

Reply:
Sorry I misspelled it 🤦‍♂️